mk/PX4/ROMFS/defaults.parm

```diff
@@ -74,7 +74,7 @@ MOT_THR_MIX_MIN, 0.150000
 MOT_THST_BAT_MAX, 16.800000
 MOT_THST_BAT_MIN, 12.000000
 MOT_THST_EXPO, 0.800000
-MOT_THST_MAX, 0.940000
+MOT_THST_MAX, 0.965000
```

Comment:
It is a characteristics of the esc and consistent for the units I tested. Given the gross correction inputs for attitude, a small amount of saturation inside the esc itself is preferable to leaving some thrust on the table.